c/tests/test_tables.c

```diff
@@ -1737,8 +1737,34 @@ test_individual_table(void)
     tsk_individual_table_print_state(&table, _devnull);
     tsk_individual_table_dump_text(&table, _devnull);
 
+    ret = tsk_individual_table_init(&table, 0);
+    CU_ASSERT_EQUAL_FATAL(ret, 0);
+    CU_ASSERT_EQUAL(table.metadata_schema_length, 0);
+    CU_ASSERT_EQUAL(table.metadata_schema, NULL);
+    const char *example = "An example of metadata schema with unicode 🎄🌳🌴🌲🎋";
```

Comment:
Eeek, can we do this? Isn't strlen going to choke on this? I don't think we need to worry about unicode explicitly at the C level --- we just need to show that arbitrary bytes (not just ascii chars) can round trip through the metadata schema field.

Reply:
`strlen("🎄")` is 4. So it all should work!

Reply:
I see - I guess by chance this unicode code point doesn't contain any embedded zeros? I'm also  surprised that the compiler tool chains are OK with having non utf-8 values in a C source file. Still, it all works so there's no point in worrying about it.